packages/next/package.json

```
@@ -73,7 +73,7 @@
   ],
   "dependencies": {},
   "peerDependencies": {
-    "@tanstack/react-query": "^5.0.0",
+    "@tanstack/react-query": ">=v5.25.0",
```

```suggestion
    "@tanstack/react-query": "^5.25.0",
```

Want to lock it at v5 major